honeycomb/src/main/java/org/commonjava/o11yphant/honeycomb/HoneycombContextualizer.java

```diff
@@ -39,6 +45,8 @@
 
     private static ThreadLocal<Span> SPAN = new ThreadLocal<>();
 
+    private static ThreadLocal<SpanContext> SPAN_CONTEXT = new ThreadLocal<>();
```

Comment:
nitpick: I think the SPAN and SPAN_CONTEXT can be set as final fields

Reply:
Ok. Updated.